[Bugfix] Fixes For Arrow Box Height - Payment Methods | Client Show Page

Civolilah opened the pull request:
@beganovich @turbo124 This PR includes fixes for arrow box height on the payment methods card under the client show page. Let me know your thoughts.

turbo124 commented:
<img width="1135" height="464" alt="image" src="https://github.com/user-attachments/assets/23b1d791-6358-4611-b620-6fb9251dd0a2" />

The problem still persists.

The badges should be aligned identically. In the example above the second payment method's badge/button is on under the credit card number...

Civolilah commented:
> <img alt="image" width="689" height="277" src="https://private-user-images.githubusercontent.com/5827962/489319984-d614bd04-3a57-4f00-8d55-c32fe7abef5f.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTc5Mjc3NDIsIm5iZiI6MTc1NzkyNzQ0MiwicGF0aCI6Ii81ODI3OTYyLzQ4OTMxOTk4NC1kNjE0YmQwNC0zYTU3LTRmMDAtOGQ1NS1jMzJmZTdhYmVmNWYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MTVUMDkxMDQyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWQxM2ExMGYzY2RlODYxMzhjODIxNzgxODFmODRhYjdhMjFiM2Y0NTZmY2EzOWYzZDVjYmI2NDVhZTMwY2I5ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.UTB9I3__qfv4ZZ337PNdR050nvz20iR3iThoaa3K87E">
> the alignment is now off vertically? it should be on the same line as the card expiry.

@turbo124 Ah okay, sorry I didn't understand your comment 100%. I fixed it, and also fixed the double "external link" icon from your screenshot. Screenshot:

<img width="422" height="238" alt="Screenshot 2025-09-15 at 11 48 13" src="https://github.com/user-attachments/assets/a98f68ea-3b70-4a95-af87-691484a6f2e1" />

Let me know your thoughts.

beganovich reviewed (approved):
Thanks 👍
